src/components/poster/Poster.js

```diff
@@ -5,14 +5,14 @@ import './Poster.css';
 const Poster = ({photo, id, title}) => {
   if (photo) {
     return (
-      <article>
+      <>
```

Comment:
Actually, since we are returning a single element (the img) we don't even need the <> </>!
Good catch tho!

Reply:
You're absolutely right, just fixed it. Thank you!